libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.kt

```
@@ -1231,7 +1231,9 @@ open class SalesforceSDKManager protected constructor(
     fun showDevSupportDialog(frontActivity: Activity?) {
         if (!isDevSupportEnabled() || frontActivity == null) return
 
-        devActionsDialog?.dismiss()
+        runCatching {
+            devActionsDialog?.dismiss()
```

This was producing a crash when the dialog dismisses on configuration change and then is shown again.  It's a safe exception to swallow.